README.md

```diff
@@ -93,45 +93,115 @@ Brainflow:
-| osc_band_power_right_alpha | Brainflow/PowerBands/Right/Alpha |
-| osc_band_power_right_beta | Brainflow/PowerBands/Right/Beta |
-| osc_band_power_right_theta | Brainflow/PowerBands/Right/Theta |
-| osc_band_power_right_delta | Brainflow/PowerBands/Right/Delta |
-| osc_band_power_right_gamma | Brainflow/PowerBands/Right/Gamma |
-| osc_band_power_avg_alpha | Brainflow/PowerBands/Average/Alpha |
-| osc_band_power_avg_beta | Brainflow/PowerBands/Average/Beta |
-| osc_band_power_avg_theta | Brainflow/PowerBands/Average/Theta |
-| osc_band_power_avg_delta | Brainflow/PowerBands/Average/Delta |
-| osc_band_power_avg_gamma | Brainflow/PowerBands/Average/Gamma |
-| osc_battery_lvl | Brainflow/Device/Battery/Level |
-| osc_is_connected | Brainflow/Device/Connected |
-| osc_time_diff | Brainflow/Device/SecondsSinceLastUpdate |
-| HueShift | Brainflow/Addon/HueShift |
+| osc_focus_left | BFI/NeuroFB/FocusLeft |
+| osc_focus_right | BFI/NeuroFB/FocusRight |
+| osc_focus_avg | BFI/NeuroFB/FocusAvg |
+| osc_relax_left | BFI/NeuroFB/RelaxLeft |
+| osc_relax_right | BFI/NeuroFB/RelaxRight |
+| osc_relax_avg | BFI/NeuroFB/RelaxAvg |
+| osc_heart_bpm | BFI/Biometrics/HeartBeatsPerMinute |
+| osc_heart_bps | BFI/Biometrics/HeartBeatsPerSecond |
+| osc_oxygen_percent | BFI/Biometrics/OxygenPercent |
+| osc_respiration_bpm | BFI/Biometrics/BreathsPerMinute |
+| osc_band_power_left_alpha | BFI/PwrBand/Left/Alpha |
+| osc_band_power_left_beta | BFI/PwrBand/Left/Beta |
+| osc_band_power_left_theta | BFI/PwrBand/Left/Theta |
+| osc_band_power_left_delta | BFI/PwrBand/Left/Delta |
+| osc_band_power_left_gamma | BFI/PwrBand/Left/Gamma |
+| osc_band_power_right_alpha | BFI/PwrBand/Right/Alpha |
+| osc_band_power_right_beta | BFI/PwrBand/Right/Beta |
+| osc_band_power_right_theta | BFI/PwrBand/Right/Theta |
+| osc_band_power_right_delta | BFI/PwrBand/Right/Delta |
+| osc_band_power_right_gamma | BFI/PwrBand/Right/Gamma |
+| osc_band_power_avg_alpha | BFI/PwrBand/Avg/Alpha |
+| osc_band_power_avg_beta | BFI/PwrBand/Avg/Beta |
+| osc_band_power_avg_theta | BFI/PwrBand/Avg/Theta |
+| osc_band_power_avg_delta | BFI/PwrBand/Avg/Delta |
+| osc_band_power_avg_gamma | BFI/PwrBand/Avg/Gamma |
+| osc_battery_lvl | BFI/Info/BatteryLevel |
+| osc_is_connected | BFI/Info/DeviceConnected |
+| osc_time_diff | BFI/Info/SecondsSinceLastUpdate |
+| HueShift | BFI/Addons/HueShift |
+
+If you still want to use the old parameter scheme, you can switch to them by adding the `--use-old-reporter` launch argument. Be aware that this will be deprecated in the future and an official announcement will be made for its sunset.
+
+## Parameter Descriptions
+### Utility
+These utility Parameters give basic information about your device and BFiVRC
+| Parameter | Description | Type |
+| ------ | ----- | ----- |
+| BFI/Info/VersionMajor | The major version number of BFiVRC | Int |
+| BFI/Info/VersionMinor | The minor version number of BFiVRC | Int |

```

Comment:
This number is a reference to the schema version not the application version, so the description for this should reference BFI instead